src/core_plugins/console/public/src/autocomplete.js

```diff
@@ -987,6 +987,7 @@ export default function (editor) {
     // change starts here
     var token = session.getTokenAt(pos.row, pos.column);
     this.base = _.clone(pos);
+    this.base.detach = () => {};
```

Comment:
Do you know if there is some cleanup we should be doing here? Why is a `noop` function sufficient? A comment in the body of the function would be nice.

Reply:
the detach hook was added as part of the new version.  I added this noop to fix a fatal issue.

Reply:
I think it's important to understand what the `detach` function on positions normally does to make sure we don't need to do something here. Just because there isn't a fatal issue doesn't mean that this works as it should.

Reply:
detach is defined elsewhere in the code to detach event listeners.  as we are not creating any here, i think it is safe.